main.py

```diff
@@ -59,16 +59,16 @@ def add_handlers(dp: Dispatcher):
 
     dp.add_handler(report_spot_conv_handler())
     # Command handlers
-    dp.add_handler(CommandHandler("start", start_cmd))
-    dp.add_handler(CommandHandler("help", help_cmd))
-    dp.add_handler(CommandHandler("rules", rules_cmd))
-    dp.add_handler(CommandHandler("stats", stats_cmd))
-    dp.add_handler(CommandHandler("settings", settings_cmd))
-    dp.add_handler(CommandHandler("sban", sban_cmd))
-    dp.add_handler(CommandHandler("clean_pending", clean_pending_cmd))
-    dp.add_handler(CommandHandler("db_backup", db_backup_cmd, run_async=True))
-    dp.add_handler(CommandHandler("purge", purge_cmd, run_async=True))
-    dp.add_handler(CommandHandler("cancel", cancel_cmd))  # it must be after the conversation handler's 'cancel'
+    dp.add_handler(CommandHandler("start", start_cmd, Filters.command))
```

Comment:
_CommandHandler_ handlers have already _Filters.command_ applied, so it is unnecessary.
Instead, it may be useful in _MessageHandler_ handlers.

Reply:
they can generate this error too